src/LagoonCommands.php

```diff
@@ -213,7 +276,11 @@ public function getLagoonEnvs() {
                     standbyAlias,
                     environments {
                     name,
-                    openshiftProjectName
+                    openshiftProjectName,
+                    openshift {
```

Comment:
Are we able to replace these with `kubernetesNamespaceName` and `kubernetes` yet? I hate to think of us writing new legacy code?

Reply:
nice